dynamorestart/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM composer:latest
```

We may want to switch this from "latest" to "2" to avoid being surprised by possibly-breaking changes in Composer 3:
```suggestion
FROM composer:2
```
As with the others, this is just a suggestion.